packages/devtools/lib/src/timeline/frame_flame_chart.dart

```diff
@@ -79,20 +75,19 @@ class FrameFlameChart extends CoreElement {
     }
 
     void drawCpuEvents() {
+      final int sectionTop = row * rowHeight;
       final CoreElement sectionTitle = div(text: 'CPU', c: 'timeline-title');
       sectionTitle.element.style.left = '0';
-      sectionTitle.element.style.top = '0';
+      sectionTitle.element.style.top = '${sectionTop}px';
       add(sectionTitle);
 
       maxRow = row;
 
-      for (TimelineEvent event in frame.cpuEvents) {
-        drawRecursively(event, row);
-      }
+      drawRecursively(frame.pipelineProduceFlow, row);
 
       row = maxRow;
 
-      row++;
+      row += 2;
```

Comment:
document why this is += 2
Did you really just want to increase rowHeight?

Reply:
No, I wanted to add additional spacing between the CPU events and GPU events. I'll add a comment or try to do that in a better way.

Reply:
done.